test/tools/ossfuzz/protoToAbiV2.cpp

```diff
@@ -690,10 +690,25 @@ void ProtoConverter::visit(TestFunction const& _x)
 		uint returnVal = this.coder_public(<parameter_names>);
 		if (returnVal != 0)
 			return returnVal;
-		return (uint(1000) + this.coder_external(<parameter_names>));
+
+		returnVal = this.coder_external(<parameter_names>);
+		if (returnVal != 0)
+			return uint(200000) + returnVal;
+
+		bytes memory argumentEncoding = abi.encode(<parameter_names>);
+
+		returnVal = checkEncodedCall(this.coder_public.selector, argumentEncoding, <invalidLengthFuzz>);
```

Comment:
Ah yes, I hadn't thought about ``.selector`` - makes a lot of sense to use for this here.

Reply:
I wasn't very sure when I took this approach, but in the end it worked out :-)